app/views/providers/date_client_told_incidents/show.html.erb

```diff
@@ -10,6 +10,8 @@
     <%= govuk_form_group show_error_if: @form.errors.present? do %>
       <%= govuk_fieldset_header page_title %>
 
+      <div class="govuk-!-padding-bottom-4"></div>
```

Comment:
The method `govuk_fieldset_header` includes an option to add padding to the bottom of this field. So rather than added in a spacing div and therefore a `div` without context, remove this and modify the `govuk_fieldset_header` call:
```
<%= govuk_fieldset_header page_title, padding_below: 4 %>
```